test/features/can_search_collection_test.rb

```diff
@@ -0,0 +1,63 @@
+require 'test_helper'
+Dir[Rails.root.join('test/**/*.rb')].each { |f| require f }
+
+class CanSearchCollectionTest < SystemTestCase
+  include Helpers::UserHelpers
+  include Helpers::ReviewsHelper
+  include Helpers::CollectionsHelper
+  include Helpers::HomeHelper
+
+  before do
+    OmniAuth.config.test_mode = true
+    mock_login(role: 'admin')
+    stub_request(:get, "#{Cmr.get_cmr_base_url}/search/collections.echo10?keyword=**&page_num=1&page_size=10&provider=LANCEAMSR2").
+        with(
+            headers: {
+                'Accept'=>'*/*',
+                'Accept-Encoding'=>'gzip;q=1.0,deflate;q=0.6,identity;q=0.3',
+                'User-Agent'=>'Ruby'
+            }).
+        to_return(status: 200, body: get_stub('search_collection_provider_ghrc.xml'), headers: {})
+    stub_request(:get, "#{Cmr.get_cmr_base_url}/search/collections.echo10?concept_id=C1996546695-GHRC_DAAC").
+        with(
+            headers: {
+                'Accept'=>'*/*',
+                'Accept-Encoding'=>'gzip;q=1.0,deflate;q=0.6,identity;q=0.3',
+                'User-Agent'=>'Ruby'
+            }).
+        to_return(status: 200, body: get_stub('search_collection_C1996546695-GHRC_DAAC.xml'), headers: {})
+    stub_request(:get, "#{Cmr.get_cmr_base_url}/search/granules.echo10?concept_id=C1996546695-GHRC_DAAC&page_num=1&page_size=10").
+      with(
+        headers: {
+          'Accept'=>'*/*',
+          'Accept-Encoding'=>'gzip;q=1.0,deflate;q=0.6,identity;q=0.3',
+          'User-Agent'=>'Ruby'
+        }).
+      to_return(status: 200, body: "<?xml version=\"1.0\" encoding=\"UTF-8\"?><results><hits>0</hits><took>8</took></results>", headers: {})
+    stub_request(:get, "#{Cmr.get_cmr_base_url}/search/collections.atom?concept_id=C1996546695-GHRC_DAAC").
+      with(
+        headers: {
+          'Accept'=>'*/*',
+          'Accept-Encoding'=>'gzip;q=1.0,deflate;q=0.6,identity;q=0.3',
+          'User-Agent'=>'Ruby'
+        }).
+      to_return(status: 200, body: get_stub("search_collection_C1996546695-GHRC_DAAC.atom"), headers: {})
+  end
+  describe 'search cmr collection' do
+    it 'search collection by provider' do
```

Comment:
Maybe name it, "search collection by GHRC provider"

Reply:
Changed